apps/web/src/components/HintsAccordion/HintsAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Heading,
+} from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { type SignPage } from "../Titles";
+
+type HintData = {
+  header?: string;
+  description?: string;
+};
+
+export const Hints: Record<SignPage, HintData> = {
+  TezSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  ContractCallSignPage: {
+    header:
+      "A confirmation takes 1 block (10 seconds). The transaction becomes final and irreversible after 2 blocks (20 seconds).",
+    description:
+      "A confirmation means that the block is confirmed by the network but the operation can still be canceled in the unlikely case of a crypto attack. Finality means that the operation is irreversible.",
+  },
+  DelegationSignPage: {
+    header: "It takes 4 cycles (~6 days) to start receiving delegation rewards.",
+    description:
+      "Delegation rewards are distributed every 2 cycles (~3 days). Ensure that the delegation fee offered by the baker is less than 100%; otherwise, you will not receive rewards. Delegation means that all your funds are delegated to one baker. The funds are still spendable, and you can cancel or change the delegation at any time.",
+  },
+  UndelegationSignPage: {
+    header:
+      "Immediately stops delegation and initiates unstaking. Restoring delegation rewards takes 4 cycles (~6 days). Unstaking takes 4 cycles (~10 days).",
```

Comment:
```suggestion
      "Immediately stops delegation and initiates unstaking. Restoring delegation rewards takes 4 cycles (~10 days). After 4 cycles (~10 days), unstaked tez can be finalized and made avaialable",
```
I need to confirm the math for the unstakes and delegation.

Also notice that in the case of a change of baker, any current liquid balance will be delegated to the new baker, but unstaked tez will keep being delegated to the old one. This is a bit of a corner case where funds end. delegated to two bakers on an overlap of a few cycles . Which is a headache. And not something you want to add to hint :)

Reply:
we can crate a web page (maybe it already exists) and link it from the hint. Then we can explain all in details including corner cases